### PR TITLE
Making importing a single file easier (Issue #5508)

### LINKED ISSUE #5508: Make it easier to import single entries
**Is your suggestion for improvement related to a problem? Please describe.**
If you import a single entry (e.g. a bib file containing only one entry), then the import dialog opens and one needs to select the entry by hand, before one can click "Import".

**Describe the solution you'd like**
Preselect the entry, so that one can directly click "Import".

### PR DESCRIPTION
**Pull request by P4trice**
Fixes #5508 
I added a condition to see if only one entry is being imported.
If the condition adheres -> mark the entry as selected.

----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

**Comment by P4trice** [comment]
I would totally agree with you. But when I try that, it doesn't seem to select the entry.
I had though to put it there too at first. Any other suggestion as to how to make it work?

**Comment by tobiasdiez** [comment]
Usually, the problem is that the code is run before the control is initialized completely. To fix this it is usually sufficient to wrap everything in a `Platform.runLater(...)` call. Can you please try this out here as well? Thanks

**Comment by P4trice** [comment]
I tried to wrap the if-statement in a Platform.runLater(..) and it did not work.
Also tried to wrap all of 'initialze' in a Platform.runLater with and without the if-statement in it.
It did not select the single file being imported.
Do you have any suggestions, as to what I could do differently? 
Sorry for the inconvenience of this PR.

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java [hunk 1]
```diff
@@ -119,6 +119,16 @@ private void initialize() {
...
 
+                    /*
+                    inserted the if-statement here, since a Platforn.runLater() call did not work.
+                    also tried to move it to the end of the initialize method, but it did not select the entry.
+                    */
+                    if (entriesListView.getItems().size() == 1) {
+                        selectAllNewEntries();
+                      }
+  
+                      return container;
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java]: Now you have two return statements 🦊

Comment (P4trice) [reply src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java]: I am so sorry for all the inconveniences. I'll correct it immediately.

**Pull request merged**